services/inventory-service/Dockerfile

```
@@ -0,0 +1,31 @@
+
+# Copy package files
+COPY services/inventory-service/package*.json ./
+RUN npm install
+
+# Copy source and build
+COPY services/inventory-service/ .
+# Build và thực hiện prune --omit=dev để giảm kích thước image, chỉ giữ lại production dependencies
+RUN npm run build && npm prune --omit=dev
+
+# Stage 2: Runtime - Sử dụng gcr.io/distroless/nodejs22 để tối thiểu hóa bề mặt tấn công
+FROM gcr.io/distroless/nodejs22-debian12:latest
```

Hadolint (2.14.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)